src/app/components/client/ExposureCard.tsx

```diff
@@ -137,6 +137,10 @@ const ScanResultCard = (props: ScanResultCardProps) => {
     );
   }
 
+  const COMPANY_NAME_MAX_CHARACTER_COUNT = 20;
```

Comment:
Question: Since the number of characters highly depends on the container width, would it make sense to take it into account as well?

Reply:
Just FYI: Saw on `main` that `makeFontSmaller` seems is being applied to all viewports.

Reply:
No worries — thanks for the follow-up!